packages/utilities/psammead-assets/src/amp-boilerplate.jsx

```diff
@@ -1,6 +1,27 @@
 /*
   AMP Boilerplate Code https://github.com/ampproject/amphtml/blob/master/spec/amp-boilerplate.md
 */
+import React from 'react';
 
 export const AMP_SCRIPT = `body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}`;
 export const AMP_NO_SCRIPT = `body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}`;
+export const AMP_JS = (
```

Comment:
Can you split these up please? so theyre 4 separate exports

Reply:
Don't you think we should keep them together? They seem to always be used as one.

Reply:
In simorgh yes, but we aim to be reusable :) someone may want to use this for the amp-analytics tag, but not use amp-consent etc :)